Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -92,6 +108,14 @@ public class RocksDBStore : BaseStore
             _blockCache = new LruCache<HashDigest<SHA256>, BlockDigest>(capacity: blockCacheSize);
 
             _path = path;
+            _txAtEachEpoch = txAtEachEpoch > 0
+                ? txAtEachEpoch
+                : throw new ArgumentException(
+                    $"{nameof(RocksDBStore)}.txAtEachEpoch cannot be 0 or less.");
+            _blockAtEachEpoch = blockAtEachEpoch > 0
+                ? blockAtEachEpoch
+                : throw new ArgumentException(
+                    $"{nameof(RocksDBStore)}.blockAtEachEpoch cannot be 0 or less.");
```

Comment:
We can make good use of [`ArgumentException()`'s `paramName` parameter](https://docs.microsoft.com/en-us/dotnet/api/system.argumentexception.-ctor?view=netstandard-2.0#System_ArgumentException__ctor_System_String_System_String_):

```suggestion
                : throw new ArgumentException(
                    "It must be greater than 0.",
                    nameof(blockAtEachEpoch));
```